js/save.js

```diff
@@ -67,19 +87,42 @@ function importFlowGram(file) {
     }
 }
 
-function openFromFile(file) {
+function importFunction(file) {
+    try {
+        const parsedFile = JSON.retrocycle(JSON.parse(file))
+        if (parsedFile.signingKey !== signingKey) {
+            swal("Flowgramming doesn't support that file!")
+            return
+        }
+        Object.keys(parsedFile.contexts).forEach((functionName) => {
+            addFunctionToTable(functionName)
+        })
+        parentWindowContexts = {
+            ...parentWindowContexts,
+            ...parsedFile.contexts,
+        }
+        window.opener.addNewContext(parsedFile.contexts)
+    } catch (e) {
+        swal('Something went wrong!')
+    } finally {
+        hideLoader()
+    }
+}
```

Comment:
When I import a whole .fgmin from function manager which contains main also, I need to define main again which is exactly how it should work. After writing main again, the program won't run.
![image](https://user-images.githubusercontent.com/41717568/132216878-04ffc173-c875-4bd0-b78e-f7a03febdcb4.png)